frontend/app/src/components/price-manager/latest/LatestPriceTable.vue

```diff
@@ -184,20 +185,15 @@ const fetchLatestPrices = async () => {
 
 const refresh = async () => {
   await fetchLatestPrices();
+  await refreshPrices(false, get(assets));
 };
 
 const { getAssetPrice } = useBalancePricesStore();
 
 const assets: ComputedRef<string[]> = computed(() => {
-  return get(latestPrices)
-    .map(({ toAsset }) => toAsset)
-    .filter(asset => asset !== CURRENCY_USD);
-});
-
-watch(assets, async assets => {
-  if (assets.length > 0) {
-    await refreshPrices(false, [...assets]);
-  }
+  return flatten(
```

Comment:
Is there any easy way to achieve the same result without relying to lodash? I am still thinking about it but it maybe would make sense to start removing it.